inventory/rh-sso/group_vars/all.yml

```diff
@@ -14,3 +14,7 @@ rhsm_repos:
 rhsm_username: "{{ lookup('env', 'RHSM_USER' )}}"
 rhsm_password: "{{ lookup('env', 'RHSM_PASSWD' )}}"
 rhsm_pool: "{{ lookup('env', 'RHSM_POOL' )}}"
+
+# Red Hat Single Sign-On Instance
+id_mgmt_sso_host: "192.168.16.3"
```

Comment:
@BinaryDevotee Any reason for the switch to id_mgmt_sso_host? We should be consistent with this if this is something that we're going to use.

Reply:
Yes, I had a few concerns that made me change this variable here.
Please, correct if I am wrong and I am happy to change this as needed:

Because we are using the same inventory `rh-sso` for both roles (identity-management and config-rh-sso), the original name of this variable is defined in `defaults/main.yml` on `config-rh-sso` role, which gathers the internel IPv4 of the server through Ansible facts and, by using the same variable name here, I foresee it breaking the automation that provisions Red Hat SSO in the first place, as it relies on the internal IP (defined by the `rh_sso_host` variable) to correctly start the SSO service because the variables defined on defaults/main.yml will get overridden by the ones defined in the inventory by the time of execution.

Two potential solutions to this situations were:
- Change the variable name in the inventory;
- Move the contents from `defaults/main.yml` to `vars/main.yml` so that it doesn't get overridden by the one defined in the inventory;

Again, please, correct if I am wrong here and I am curious to hear your thoughts on this topic.

Reply:
I reverted the variable name to the desired state, however, I will need to raise another minor PR against `config-rh-sso` role and change how the URL there is being handled.